driver-core/pom.xml

```diff
@@ -334,6 +334,13 @@
               <classpathDependencyExcludes>
                 <classpathDependencyExcludes>io.netty:netty-transport-native-epoll</classpathDependencyExcludes>
               </classpathDependencyExcludes>
+              <properties>
```

Comment:
Good point, I'll follow that example in driver-extras as I think it will work very well.